libdevice/imf/imf_fp32_dl.cpp

```diff
@@ -0,0 +1,16 @@
+//==--------------------------- imf_fp32_dl.cpp -  -------------------------==//
```

Comment:
Some description after `imf_fp32_dl.cpp -` looks missed

Reply:
Or closer - see the 1st line in some random file in `libdevice/imf` folder: https://github.com/intel/llvm/blob/sycl/libdevice/imf/imf_inline_bf16.cpp#L1

Reply:
Hi, @dm-vodopyanov 
Added the small description, could you help review again?
Thanks very much.

Reply:
@jinge90 there should be a single line which starts with `//==` and ends with `==//`
https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
Hi, @dm-vodopyanov 
Just adjusted the file header.
Thanks very much.